### PR TITLE
Adding documentation for research articles that used TARDIS

### PR DESCRIPTION
**Pull request by KevinCawley**
Added a notebook that myself and Isaac created that makes a list of papers that use TARDIS.

## Description
Myself and Isaac created ads.ipynb. This notebook uses the ads module and generates a list of papers that use TARDIS in research in a citation style described to us. The folder, 'research_done_using_TARDIS', was added to tardis/docs/research. Inside the folder is the ads.ipynb notebook that generates the list. This notebook doesn't have the ads key, the user must put their own into it to get it to work. One can be generated at https://ui.adsabs.harvard.edu/user/settings/token.
Modification to the index.rst file in tardis/docs/research to include our folder as well.

## Motivation and Context
We want to see how TARDIS is being used by researchers and give them a place to present their research that has used TARDIS.

## How Has This Been Tested?
It generates the .rst file correctly, and all the papers inside meet the criteria described to us and have been approved as using TARDIS.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have assigned/requested two reviewers for this pull request.

### CODE REVIEW

File: docs/research/research_done_using_TARDIS/ads.ipynb [hunk 1]
```diff
@@ -0,0 +1,918 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d = {'Authors': get_authors_formatted(authors), 'Year': year_list, 'Journal': get_journal(bibstems), \n",
+    "     'Title': get_titles_formatted(titles), 'Link': get_url(bibcodes)}\n",
+    "df = pd.DataFrame(data=d)\n",
+    "#df.to_csv('adslist.csv')"
```
Comment (andrewfullard) [inline docs/research/research_done_using_TARDIS/ads.ipynb:146]: Remove commented lines

File: docs/research/research_done_using_TARDIS/ads.ipynb [hunk 2]
```diff
@@ -0,0 +1,918 @@
...
+       "41  https://ui.adsabs.harvard.edu/abs/2014MNRAS.44...  "
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df"
```
Comment (andrewfullard) [inline docs/research/research_done_using_TARDIS/ads.ipynb:886]: I don't think we need multiple outputs from the notebook

File: docs/research/research_done_using_TARDIS/ads.ipynb [hunk 3]
```diff
@@ -0,0 +1,918 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_journal(bibstems):\n",
+    "    \"\"\"This function takes the list of bibstems and returns the journal that they are in, formatted.\"\"\"\n",
```
Comment (andrewfullard) [inline docs/research/research_done_using_TARDIS/ads.ipynb:64]: You do not need to reformat the bibstem into a full journal name. TARDIS users will know what the journal abbreviations mean and can look them up via NASA ADS if not

Comment (KevinCawley) [reply docs/research/research_done_using_TARDIS/ads.ipynb:64]: I understand that, but as we have already formatted the bibstems is it fine to leave it like this, or do you want it changed so it doesn't display the whole journal?

File: docs/research/research_done_using_TARDIS/ads.ipynb (hunk 2 above)
Comment (KevinCawley) [reply docs/research/research_done_using_TARDIS/ads.ipynb:885]: Would it be better to keep the string output that it displays inside of the notebook and remove the dataframe, or remove the string output that it displays inside of the notebook and keep the dataframe?

**Review by andrewfullard — APPROVED** [review]

**Pull request merged**